tutorials/printing.adoc

```diff
@@ -1,83 +1,218 @@
-Tisk
-----
+= Tisk
+Marián Hlaváč <marian.hlavac@fit.cvut.cz>
+:toc:
 
-OctoPrint
-~~~~~~~~~
+Tento týden se konečně dostaneme k tomu, na co jsme se všichni jistě celou dobu těšili. 
+Využijeme získané teoretické znalosti a uplatníme je prakticky.
 
-Při kalibračním a dalším tisku budeme používat webový ovládací interface
-http://octoprint.org[OctoPrint].
+Seznámíme se s tiskárnou, vysvětlíme si, k čemu jsou určeny její jednotlivé části a připravíme ji k tisku. 
+Seznámíme se s nástroji, kterými budeme přenášet tisková data a spouštět tisk a závěrečně na tiskárnách něco vytiskneme.
+ 
+_Pokud již 3D tiskárny znáte, můžete některé kapitoly v textu vynechat, ale nevynechávejte kapitolu <<Příprava tiskárny>>, je povinná pro všechny. Až pak můžete pokračovat čtením <<Zadání úkolů, zadaných ukolů>>._
 
-image:../images/printing/octoprint.png[Náhled]
 
-(Obrázek z
-https://github.com/foosel/OctoPrint/blob/master/README.md[README]
-OctoPrintu © Gina Häußge (AGPL).)
+== Seznámení se s tiskárnou
 
-Předletová příprava
-~~~~~~~~~~~~~~~~~~~
+Na stole před vámi by měla ležet FDM tiskárna http://reprap4u.cz/rebelix/[_RebeliX_] a jedna či dvě špule materiálu ABS.
 
-Před spuštěním tisku je potřeba dodržet několik kroků. V případě
-tiskárnu nezkontrolujeme, můžeme skončit s špatným výtiskem, nebo v
-horším případě s poškozenou tiskárnou.
+Na fotografii níže se seznamte s pojmy, které budou dále ve studijním textu zmíněny.
 
-1.  Zkontrolovat správné nastavení osy Z (výška endstopu)
-2.  Připravit tiskovou desku (bude upřesněno)
-3.  Předehřát tiskovou desku (trvá déle než předehřátí trysky)
+.Tiskána RebeliX
+image::../images/printing/rebelix-photo-annotated.jpg[Rebelix, width=720, link=../images/printing/rebelix-photo-annotated.jpg]
```

Comment:
I'd prefer to use `:imagesdir: ../images/printing/` at the top and simply `image::rebelix-photo-annotated.jpg[...]` everywhere else.